src/common/api/types.ts

```diff
@@ -1,3 +1,20 @@
+/**
```

Comment:
Same as on `swagger.yaml`, let's only keep those changes that are relevant to this ticket.

Reply:
True. I'll change to put only changes relevant for this PR